client/src/pages/HomePage.js

```diff
@@ -31,4 +34,10 @@ const LogoWrapper = styled.div`
   flex-direction: column;
   text-align: center;
   color: var(--contrast_color);
+  align-items: center;
```

Comment:
If you want to have all content centered-centered here, you could try "display: grid, place-items: center" and remove "display: flex, justify-content, flex-direction, align-items"